yearn/apy/v2.py

```diff
@@ -68,9 +67,28 @@ def simple(vault, samples: ApySamples) -> Apy:
     apys = [week_ago_apy, month_ago_apy, inception_apy]
     net_apy = next((value for value in apys if value != 0), 0)
 
-    # performance fee is doubled since 1x strategists + 1x treasury
-    performance = (contract.performanceFee() * 2) if hasattr(contract, "performanceFee") else 0
+    # for performance fee, half comes from strategy (strategist share) and half from the vault (treasury share)
+    strategy_fees = []
+    for x in range(20): # we can have up to 20 strategies
```

Comment:
Consider using `Vault.strategies`, which should be more complete view than reading the withdrawal queue.